### ISSUE TITLE
Additional feature

### ISSUE THREAD

**Issue by JSBOfficial** [SuiMachine/DFBHD-Wrapper#1]
Hi! Can I ask for your help for the RTX Remix community about the ongoing problem we have on the culling and shader problem? The shader problem is only problematic for the lod0 but works perfectly if it can be forced to be lod1 for all distance.

**Comment by SuiMachine** [comment]
Nah, sorry - I am not even interested into looking into this with the amount of things I do lately.

**Comment by JSBOfficial** [comment]
Thank you for replying.

**Comment by justtoask** [comment]
I'd recommend closing this issue as not planned btw
